cmd/mattermost/commands/command.go

```diff
@@ -148,6 +158,24 @@ func createCommandCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func showCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	slashCommand := getCommandFromCommandArg(a, args[0])
+	if slashCommand == nil {
+		command.SilenceUsage = true
+		return errors.New("Unable to find command '" + args[0] + "'")
+	}
+	// pretty print
+	fmt.Printf("%s", prettyPrint(configToMap(*slashCommand)))
```

Comment:
I'm not sure about this approach, at least, this approach directly. You are using two functions designed to be used with the config. That functions works properly, but they are named and located in a way that is not obvious this usage.

Here I see two options:

1. Move the `prettyPrint` and `structToMap` to an utils.go file, and rename `prettyPrint` to `prettyPrintMap`, and then use them here (do not use `configToMap`, because `structToMap` do the same and have a better naming. Maybe you can create a `prettyPrintStruct` function to combine both directly.
2. Remove this and use ad-hoc implementation for printing the command.

Reply:
Thanks for pointing out how those printing functions were located and used. I definitely agree that it becomes confusing using them directly from config like I previously was.

I pushed up rebased changes that includes the design you outlined in option 1. I made the `utils.go` file directly in the `commands` package. If it should be moved to the `utils` package let me know and I will move it again.

Please let me know what you think.

Reply:
Awesome, looks pretty good. One more thing, you moved the `config.go` functions to the `utils.go` file, but you didn't move the `config_test.go` tests related to that functions to a new `utils_test.go` file.

Reply:
Awesome, thanks. Now looks good to me.